extra/resources/ClusterMon.in

```diff
@@ -63,10 +63,13 @@ as seconds.
 
 <parameter name="extra_options">
 <longdesc lang="en">
-Additional options to pass to crm_mon.  Eg. -n -r
+Additional options to pass to crm_mon.  Allowed values are "-n", "-r", TODO: what else?.
 </longdesc>
 <shortdesc lang="en">Extra options</shortdesc>
 <content type="string" default="" />
+  <option value="-n" />
+  <option value="-r" />
+</content>
```

Comment:
This should be left as it was -- this can contain any arguments supported by crm_mon, which can include arbitrary strings (resource names, etc.).

(That will also fix CI error, which is due to content being closed twice here)